support/ci/deploy_mac_launcher.sh

```diff
@@ -8,14 +8,14 @@ version="$(cat VERSION)"
 mac_builder=admin@74.80.245.236
 
 BINTRAY_REPO=unstable
-if [ $version == "$TRAVIS_TAG" ]; then
+if [ "$version" == "$TRAVIS_TAG" ]; then
   BINTRAY_REPO=stable
 fi
 
 # kick off the mac unstable build
 echo "Kicking off the $BINTRAY_REPO mac build"
 var_file=/tmp/our-awesome-vars
-ssh_args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /tmp/habitat-srv-admin"
+ssh_args=(-o "StrictHostKeyChecking=no" -o "UserKnownHostsFile=/dev/null" -i /tmp/habitat-srv-admin)
```

Comment:
It's necessary to change this to an array so that it can be appropriately quoted later. Left as a unquoted scalar, it's susceptible to undesired re-spltting and globbing. As a quoted scalar, it wouldn't be functional since the entire contents would be passed to `ssh` (and `scp`) as a single arg. As an array, we can pass `"${ssh_args[@]}"`, which will be split into separate arguments, without the arguments themselves being subject to resplitting or globbing.